Update README.md (#468)
okta/okta-sdk-dotnet: README.md

});
```

## Manual pagination

> Collections can be fetched with manually controlled pagination as in code sample below.

Reviewer: ```suggestion
Collections can be fetched with manually controlled pagination, see the following.
```

Reviewer: Depending on how this page gets rendered, the code may not be "below".
